README.md

```diff
@@ -1,31 +1,42 @@
 # Galvanize Q3 Group Project Proposal
 
-1. Fork/clone this repo
-2. Update this readme with proposal
-  * Use valid markdown formatting!
-3. Pull request
-
-## Group Members
-
-
-## Project Description
-
-
-## Who uses it?
-
-
-## What outputs do they need?
-
-
-## What inputs are needed to generate those outputs?
-
-
-## What technologies will you use that weren't covered in class?
-
-
-## What technologies do you plan to use?
-
-
-## Feature list
-
-
+### Team Name: __ODBzzz__
+
+### Group Members:
+- George Flauta
+- Keith Davis
+- Lee Robinson
+- Thomas Jones
+
+### Project/Application Name: __Testing, Testing__
+
+### Project Description:
+As may be inferred by the name, _Testing, Testing_ is an educational platform masked as a challenge-based web, game app.  Using a computer interface, players would pit their knowledge of a subject or ability to perform a task against other users.   
+
+### Who uses it?
+The MVP audience for _Testing, Testing_ will be developers, who should be sufficiently familiar with the included code logic and TDD (Test Drive Design).  Beyond proof-of-concept, content for the _Testing, Testing_ platform will be increased to facilitate training or educational opportunities (e.g. schools, business training, friendly competition, etc)
+
+### What outputs do they need?
+Users will need to write (or correct) code to successfully pass TDD tests in order to progress.  Winners will receive visual confirmation of either winning or losing a challenge.  Additionally, winning a challenge will provide (dispense) a physical reward (details TBD).
+
+### What inputs are needed to generate those outputs?
+As a web-based application, access to the Internet will be necessary; players will use a keyboard as the primary source of input.  An understanding of TDD would be helpful, but test failure codes will be provided to users.   
+
+### What technologies do you plan to use?
+- Arduino
+- Raspberry Pi
+- Photon wifi / ITFFF
+
+### Libraries/frameworks used:
+- TDD (Test Drive Design), Mocha/Chai
+- AngularJs, Javascript
+- Arduino_Machine_Language
+
+### Feature list
+- MVP
+  1. Remote trigger for delivery (or dispensing) of reward.
+  2. Develop front-end condition to trigger delivery system through completing or winning challenge.
```

Comment:
Could you include more than just the remote candy delivery system for the features? I know you are going to do more than just build that.